lib/new_relic/agent/instrumentation/ruby_openai.rb

```diff
@@ -12,9 +12,10 @@
   OPENAI_VERSION = Gem::Version.new(OpenAI::VERSION) if defined?(OpenAI)
 
   depends_on do
-    # add a config check for ai_monitoring.enabled
-    # maybe add DT check here eventually?
-    defined?(OpenAI) && defined?(OpenAI::Client) &&
+    NewRelic::Agent.config[:'ai_monitoring.enabled'] != false &&
+      NewRelic::Agent.config[:'instrumentation.ruby_openai'] != 'disabled' &&
+      # maybe add DT check here eventually?
```

Comment:
I think we can safely remove this comment based on today's [DT update](https://newrelic.slack.com/archives/C05DWELBP0F/p1708098719787269) (internal)

Reply:
Removed! https://github.com/newrelic/newrelic-ruby-agent/pull/2443/commits/95e5d803ee09c0e69fb2db7f4fc673a0df46a691